tools/base/test/define-element.test.ts

```diff
@@ -0,0 +1,213 @@
+    'sp-radio-group': () =>
+        import('@spectrum-web-components/radio/sp-radio-group.js'),
+    'sp-radio': () => import('@spectrum-web-components/radio/sp-radio.js'),
+    'sp-search': () => import('@spectrum-web-components/search/sp-search.js'),
+    'sp-sidenav-item': () =>
+        import('@spectrum-web-components/sidenav/sp-sidenav-item.js'),
+    'sp-sidenav': () =>
+        import('@spectrum-web-components/sidenav/sp-sidenav.js'),
+    'sp-slider': () => import('@spectrum-web-components/slider/sp-slider.js'),
+    'sp-slider-handle': () =>
+        import('@spectrum-web-components/slider/sp-slider-handle.js'),
+    'sp-split-button': () =>
+        import('@spectrum-web-components/split-button/sp-split-button.js'),
+    'sp-split-view': () =>
+        import('@spectrum-web-components/split-view/sp-split-view.js'),
+    'sp-status-light': () =>
+        import('@spectrum-web-components/status-light/sp-status-light.js'),
+    'sp-swatch-group': () =>
+        import('@spectrum-web-components/swatch/sp-swatch-group.js'),
+    'sp-swatch': () => import('@spectrum-web-components/swatch/sp-swatch.js'),
+    'sp-switch': () => import('@spectrum-web-components/switch/sp-switch.js'),
+    'sp-table': () => import('@spectrum-web-components/table/sp-table.js'),
+    'sp-table-body': () =>
+        import('@spectrum-web-components/table/sp-table-body.js'),
+    'sp-table-cell': () =>
+        import('@spectrum-web-components/table/sp-table-cell.js'),
+    'sp-table-head': () =>
+        import('@spectrum-web-components/table/sp-table-head.js'),
+    'sp-table-checkbox-cell': () =>
+        import('@spectrum-web-components/table/sp-table-checkbox-cell.js'),
+    'sp-table-head-cell': () =>
+        import('@spectrum-web-components/table/sp-table-head-cell.js'),
+    'sp-table-row': () =>
+        import('@spectrum-web-components/table/sp-table-row.js'),
+    'sp-tab': () => import('@spectrum-web-components/tabs/sp-tab.js'),
+    'sp-tabs': () => import('@spectrum-web-components/tabs/sp-tabs.js'),
+    'sp-tabs-overflow': () =>
+        import('@spectrum-web-components/tabs/sp-tabs-overflow.js'),
+    'sp-tag': () => import('@spectrum-web-components/tags/sp-tag.js'),
+    'sp-tags': () => import('@spectrum-web-components/tags/sp-tags.js'),
+    'sp-textfield': () =>
+        import('@spectrum-web-components/textfield/sp-textfield.js'),
+    'sp-thumbnail': () =>
+        import('@spectrum-web-components/thumbnail/sp-thumbnail.js'),
+    'sp-toast': () => import('@spectrum-web-components/toast/sp-toast.js'),
+    'sp-tooltip': () =>
+        import('@spectrum-web-components/tooltip/sp-tooltip.js'),
+    'sp-top-nav': () =>
+        import('@spectrum-web-components/top-nav/sp-top-nav.js'),
+    'sp-tray': () => import('@spectrum-web-components/tray/sp-tray.js'),
+    'sp-underlay': () =>
+        import('@spectrum-web-components/underlay/sp-underlay.js'),
+};
+
+const browser: 'webkit' | 'firefox' | 'chromium' = isWebKit()
```

Comment:
Browser knowledge is only to differentiate error message, right?

Reply:
Right.